api/base/serializers.py

```diff
@@ -29,6 +29,65 @@ def _url_val(val, obj, serializer, **kwargs):
         return val
 
 
+class HyperLinkedIdentityFieldWithMeta(ser.HyperlinkedIdentityField):
+
+    def __init__(self, view_name=None, **kwargs):
```

Comment:
I overrode the three methods that I needed to change.
1) __init__ because meta information is now an argument
2) get_url because I had to add the lines 
```
if lookup_value is None:
     return None
```
This was when there was no parent for example, to prevent the parent link from returning `http://localhost:8000/v2/nodes/None.
3) to_representation to return link as a list [url, meta_info]